src/price/Sums.php

```diff
@@ -6,6 +6,9 @@
 {
     /**
      * @param int[]|null $values quantity => total sum for the quantity
+     * Quantity of what?
+     * Sum of what?
+     * If you know answers please write in the comment
```

Comment:
> Quantity of what?

Of the action being charges

> Sum of what?

Amount for this given Quantity


If used to denote bulk prices.
E.g. when you buy an SSL certificate for 1 year – it costs 10$
But for 2 years you pay 15$.

It is stored as 

```php
[ 1 => 10, 2 => 15 ]
```